cl_dll/cdll_int.cpp

```diff
@@ -217,6 +222,42 @@ int CL_DLLEXPORT HUD_Redraw( float time, int intermission )
 {
 //	RecClHudRedraw(time, intermission);
 
+/* megasausageking: custom crosshair drawing happens here */
+	bool bdrawcross = (CVAR_GET_FLOAT("cl_customcrosshair") == 1);
+	bool bdrawcircle = (CVAR_GET_FLOAT("cl_cross_circle") == 1);
+	float offsetX = CVAR_GET_FLOAT("cl_cross_x");
+	float offsetY = CVAR_GET_FLOAT("cl_cross_y");
+	float radius = CVAR_GET_FLOAT("cl_cross_circle_radius");
+	float circlewidth = CVAR_GET_FLOAT("cl_cross_circle_width");
+	float width = CVAR_GET_FLOAT("cl_cross_width");
+	float size = CVAR_GET_FLOAT("cl_cross_size");
+	float red = CVAR_GET_FLOAT("cl_cross_red");
+	float green = CVAR_GET_FLOAT("cl_cross_green");
+	float blue = CVAR_GET_FLOAT("cl_cross_blue");
+	float crossalpha = CVAR_GET_FLOAT("cl_cross_alpha");
+	float circlealpha = CVAR_GET_FLOAT("cl_cross_circle_alpha");
+	float gap = CVAR_GET_FLOAT("cl_cross_gap");
+	float dotsize = CVAR_GET_FLOAT("cl_cross_dot_size");
+	bool bdrawdot = (CVAR_GET_FLOAT("cl_cross_dot") == 1);
```

Comment:
I'd store the cvars in variables and access their values with ->value, that's faster than doing string lookup 20 times every drawn frame.